package.json

```diff
@@ -130,7 +130,7 @@
 		"test": "npm run lint && npm run test-unit",
 		"test-php": "npm run lint-php && npm run test-unit-php",
 		"ci": "concurrently \"npm run lint && npm run build\" \"npm run test-unit:coverage-ci\"",
-		"fixtures:clean": "rimraf \"blocks/test/fixtures/*.+(json|serialized.html)\"",
+		"fixtures:clean": "rimraf \"core-blocks/test/fixtures/*.+(json|serialized.html)\"",
```

Comment:
I thought I fixed it in one of the PRs when having an issue with those fixtures.